spec/components/glance_metric_spec.rb

```diff
@@ -23,8 +23,9 @@
   end
 
   it "does not render if figure is not supplied" do
```

Comment:
Please update the test to say it renders "no data" as it used to render nothing and if this is failing it will be misleading trying to figure out what to fix.

Reply:
Good call, I've updated both tests 👍